### ISSUE TITLE
Add SSL support

### ISSUE THREAD

**Issue by nikipore** [nikipore/stompest#3]
The URI scheme supports only TCP, no SSL (the author doesn't need it because the client is run in "safe" production environments). For the async client, it should be straightforward to support SSL by means of the Endpoint API. Contributions are welcome!

**Comment by zachjacobs** [comment]
Was this ever added?  I'm guessing not since I get this when trying to use ssl:

```python
config = StompConfig('ssl://%s:%d' % (host, port), login=user, passcode=password, version='1.1')
```

and output:
```
Traceback (most recent call last):
  File "publisher.py", line 41, in <module>
    client = Stomp(config)
  File "build/bdist.macosx-10.11-intel/egg/stompest/sync/client.py", line 51, in __init__
  File "build/bdist.macosx-10.11-intel/egg/stompest/protocol/failover.py", line 43, in __init__
  File "build/bdist.macosx-10.11-intel/egg/stompest/protocol/failover.py", line 163, in __init__
  File "build/bdist.macosx-10.11-intel/egg/stompest/protocol/failover.py", line 189, in _parse
ValueError: invalid uri: ssl://localhost:61613 [invalid broker(s): 'NoneType' object has no attribute 'groupdict']
```

**Comment by hhamalai** [comment]
This issue still open and no SSL support yet.

**Comment by nikipore** [comment]
Sorry, @joshainglis. I had to revert to version 2.1.6 since you haven't updated the documentation yet. My acceptance of your pull request was somewhat premature; when I merged, the tests were still broken etc. -- I'm not a git wiz.

Since the [Python 3 pull request](https://github.com/nikipore/stompest/pull/34) is based on that latest PyPI version, I prefer to accept these changes first. I will gladly accept your rebased pull request when the docs are complete.

**Comment by nikipore** [comment]
Now we're on version 2.2.0 which speaks both Python 2.7 and Python 3, and we might attempt to integrate SSL support again if you're still interested.

**Comment by bspink** [comment]
@nikipore Hi there, I'd be happy to take over the SSL support PR as @joshainglis has left the company we're using this client at, and I've just come across this now.

**Comment by nikipore** [comment]
Yes, please feel free to take over. I was already wondering why @joshainglis did not carry on. I purged the ongoing pull request because of the Python 3 upgrade (which means you have to rebase and start over), but I also dropped Python 2.6 support (which makes life easier for you).

**Comment by bspink** [comment]
@nikipore understood, I should be able to look at it in the next couple of weeks. 👍

**Comment by gwilliams-tripwire** [comment]
@bspink Are you still intending to finish the SSL support PR work?  I also have a need for SSL support, so I'm interested in the status of this.

**Comment by nikipore** [comment]
Included in release 2.3.0.
